lib/commands/file-movement.js

```diff
@@ -269,6 +295,52 @@ async function pullFromRealDevice (device, remotePath, isFile) {
   }
 }
 
+async function createMountDirForDocuments (mntRoot) {
+  const p = path.posix.join(mntRoot, 'Documents');
```

Comment:
`path.posix` makes sense only for paths on the mobile device, but not for local paths